assets/src/blocks/Spreadsheet/SpreadsheetBlock.js

```diff
@@ -0,0 +1,62 @@
+import {SpreadsheetEditor} from './SpreadsheetEditor';
+import {frontendRendered} from '../../frontendRendered';
+import {SpreadsheetFrontend} from './SpreadsheetFrontend';
+import {renderToString} from 'react-dom/server';
+
+const BLOCK_NAME = 'planet4-blocks/spreadsheet';
+
+const attributes = {
+  url: {
+    type: 'string',
+    default: '',
+  },
+  color: {
+    type: 'string',
+    default: 'grey',
```

Comment:
Do you think if it's possible to use p4 colors here?

Reply:
This name is just an alias to tell the editor which color is selected by default and not the actual color itself

Reply:
Does it come from `theme.json` then?

Reply:
These are just for the classnames that are generated for the table color, such as `is-color-blue`, it's not related to our palette